scripts/pricefeeds/pricefeeds.py

```diff
@@ -128,7 +138,13 @@ def fetch_from_wallet(rpc):
             quote_precision = assets[quote["asset_id"]]["precision"]
             base_price  = (int(base["amount"]) / 10 ** base_precision)
             quote_price = (int(quote["amount"]) / 10 ** quote_precision)
-            price_median_blockchain[asset] = float(base_price / quote_price)
+
+            quote_asset_id = result['current_feed']['settlement_price']['quote']['asset_id']
+            quote_asset = assets[quote_asset_id]["symbol"]
+            if quote_asset == "BTS":
+                price_median_blockchain[asset] = float(base_price / quote_price)
+            else:
+                price_median_blockchain[asset] = float(base_price / quote_price) * price_median_blockchain[quote_asset]
```

Comment:
You shouldn't do this distinction! A price is *always* define with a quote and a base and in particular a price feed is define against it's "short backing asset*. That means that the price is always base/quote and shouldn't be corrected for another quote's price. It will simply not be BTS/* .. but *BACKING-ASSET/*